src/test/java/apicurito/tests/steps/CommonSteps.java

```diff
@@ -145,15 +154,71 @@ public void createParameterOnPage(String param, String page, String isPlus, Data
                     .shouldHaveSize(1).first().shouldBe(visible).$("button").click();
-        }else {
+        } else {
             pageElement.$(section).$$("a").filter(text(aName)).shouldHaveSize(1).first().shouldBe(visible).click();
         }
         CommonUtils.fillEntityEditorForm(table);
     }
+
+    @When("create a server on {string} page")
+    public void createAServerOnPage(String page, DataTable table) {
+        By section = CommonUtils.getSectionBy("server");
+        SelenideElement pageElement = CommonUtils.getPageElement(page);
+        CommonUtils.openCollapsedSection(pageElement, section);
+
+        for (List<String> dataRow : table.cells()) {
+
+            //open editor by plus button(true) or by link (false)
+            if (Boolean.valueOf(dataRow.get(2))) {
+                pageElement.$(section).$$("icon-button").filter(attribute("type", "add"))
+                        .shouldHaveSize(1).first().shouldBe(visible).$("button").click();
+            } else {
+                pageElement.$(section).$$("a").filter(text("Add a server")).shouldHaveSize(1).first().shouldBe(visible).click();
+            }
+
+            //set server name and click Apply
+            SelenideElement editorPage = MainPageUtils.getMainPageRoot().$("#entity-editor-form");
+            editorPage.$("#serverUrl").sendKeys(dataRow.get(0));
+            CommonUtils.getButtonWithText("Apply", editorPage).click();
+
+            if (!dataRow.get(1).isEmpty()) {
+                editorPage.$("#description").sendKeys(dataRow.get(1));
+            }
+
+            CommonUtils.getButtonWithText("Save", editorPage).click();
+        }
+    }
+
+    @When("configure server variables for {string} on {string} page")
+    public void configureServerVariablesForOnPage(String url, String page, DataTable table) {
+        By section = CommonUtils.getSectionBy("server");
+        SelenideElement pageElement = CommonUtils.getPageElement(page);
+        CommonUtils.openCollapsedSection(pageElement, section);
+        SelenideElement server = pageElement.$(section).$$(By.className("url")).filter((text(url))).first().parent();
+        CommonUtils.getKebabButtonOnElement(server).click();
+        CommonUtils.getDropdownMenuItem("Edit").click();
+
+        SelenideElement variablesArea = MainPageUtils.getMainPageRoot().$("#entity-editor-form").$(By.className("server-variables"));
+        for (List<String> dataRow : table.cells()) {
+            //select right variable
+            variablesArea.$$("li").filter(text(dataRow.get(0))).first().click();
+
+            CommonUtils.getLabelWithType("text", variablesArea.$$(By.className("panel-body")).filter(visible).first()).sendKeys(dataRow.get(1));
+
+            variablesArea.$$("textarea").filter(visible).first().sendKeys(dataRow.get(2));
+
+
+        }
+        CommonUtils.getButtonWithText("Save", MainPageUtils.getMainPageRoot().$("#entity-editor-form")).click();
+    }
+
```

Comment:
I think for better code structure, these two methods could be placed in a superclass for all the page objects (make them subclasses), which can use this, so it wouldn't be necessary to pass the page as a parameter.